template/php/flake.nix

```
@@ -0,0 +1,68 @@
+{
+  description = "Example kickstart PHP application project.";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+
+  outputs = inputs @ {flake-parts, ...}:
+    flake-parts.lib.mkFlake {inherit inputs;} {
+      systems = ["x86_64-linux" "aarch64-linux" "aarch64-darwin" "x86_64-darwin"];
+
+      perSystem = {
+        config,
+        self',
+        inputs',
+        pkgs,
+        system,
+        ...
+      }: let
+        inherit (pkgs) dockerTools php83;
+        inherit (dockerTools) buildImage;
+        inherit (php83) buildComposerProject;
+        name = "example";
+        version = "0.1.0";
+        myPhpVersion = php83.buildEnv {
```

```suggestion
        php = php83.buildEnv {
```

Let's rename this to seem less placeholder-y.

---

Renamed it to `phpEnvironment`, since it seems fair enough to use this name :D 

Also renamed the php file in the src folder to match the webserver standard of file handling (`/index.php` == `/`)